src/components/chart-scroll/ChartScroll.css

```diff
@@ -239,7 +239,30 @@ justify-content: center;
 }
 
 .csv {
-  margin-left: 1.5em;
+  
+  text-align: center;
+  margin-bottom: 2vh;
 }
 
-
+.modal {
+  background-color: white;
```

Comment:
Background color for the site is #f8f8f8

Reply:
got it, thanks.